tests/api/mod.rs

```diff
@@ -8,6 +8,7 @@ mod notifications;
 mod ping;
 mod root;
 mod search;
+mod setttings;
```

Comment:
`settings` with 3x t is on purpose?

```suggestion
mod settings;
```